spec/functional/letsencrypt_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require File.expand_path('../../spec_helper', __FILE__)
+
+ENV['ACME_KEY_1']   = '123sadaf2342rfsdf'
+ENV['ACME_TOKEN_1'] = 'fkjsdbfnkjnk354nrfwegsdfg'
+
+module Pod::TrunkApp
+  describe LetsEncryptController do
+    test_controller! LetsEncryptController
+
+    # This controller is scoped to "/well-known/" inside the app router
+
+    it 'handles ACME responses using known environment vars' do
+      get 'https://example.org/acme-challenge/' + ENV['ACME_TOKEN_1']
+      last_response.should.be.ok
+      last_response.body.should == ENV['ACME_KEY_1']
+      last_response.headers['Content-Type'].should == 'text/plain'
+    end
+
+    it '404s for responses without known environment vars' do
+      get 'https://example.org/acme-challenge/orta'
+      last_response.status.should == 404
```

Comment:
Might be worth testing that we don't leak the key here, just so we don't break it in the future?